README.md

```diff
@@ -15,6 +15,8 @@ Checkout the release notes in the [Changelog](CHANGELOG.md) page, our you can se
 
 Development guides are available in the "Android" section of the [Community Health Toolkit Docs Site](https://docs.communityhealthtoolkit.org/core/guides/android/). You will find instructions of how to setup your development environment, build and test new features, creates new branded apps, release, publish... and so on.
 
+## Settings page
+To open the app's Settings page, tap 5 times with one finger on the screen and then swipe right with two fingers, make sure to do this sequence fast.
```

Comment:
@latin-panda - can you add another sentence here please? It's not clear to me why I would want to go to this settings page, so maybe just add something like:

> Once opened, you can change the URL of the CHT server, change the color of the background and change the session timeout.  Clicking "Save" will save the changes and close the settings page.

(I made all those up, of course!)

Reply:
Done, thanks for the feedback